### DISCUSSION TITLE
Feature Request: Accessibility aria-label/aria-labelledby support

### DISCUSSION THREAD

**Discussion by AhmerMH** [ueberdosis/tiptap#4269] [Feature Requests]
### Description

ProseMirror already supports the aria-label tag by passing the `label` property in `options`. The documentation can be found here: https://prosemirror.net/docs/ref/version/0.5.0.html#label

Please add an option to pass this `label` property from tiptap, so that aria-label tag can be added to the ProseMirror div.

### Use Case

The feature helps in the accessibility of the web application and a common use case required by every web page.

### Type

Other

**Comment by svenhanssen** [comment]
You can change the DOM attributes of the element like this:

```ts
useEditor({
  editorProps: {
    attributes: {
      'aria-label': 'Custom aria label',
    },
  },
}
```
